### PR TITLE
Fix for issue #4696

### PR DESCRIPTION
**Pull request by eliangidoni**

### CODE REVIEW

**Comment by danielmewes** [comment]
Thanks! Actually @mlucy mentioned in https://github.com/rethinkdb/rethinkdb/issues/4696#issuecomment-131298227 that he already had a fix for this.

@mlucy could you take a look at this please?

File: admin/static/js/reql_docs.js [hunk 1]
```diff
@@ -31,8 +31,8 @@ reql_docs = {
     "url": "add"
   }, 
   "api/javascript/and/": {
-    "body": "bool.and(bool[, bool, ...]) &rarr; boolr.and(bool[, bool, ...]) &rarr; bool",
```
Comment (danielmewes) [inline admin/static/js/reql_docs.js]: This file is auto generated. These changes should be made in the corresponding files in https://github.com/rethinkdb/docs/tree/master/api

File: drivers/python/rethinkdb/docs.py [hunk 2]
```diff
@@ -121,8 +121,8 @@
...
-	(rethinkdb.ast.RqlQuery.__and__, b'bool & bool -> bool\nr.and_(bool, bool) -> bool\nbool.and_(bool) -> bool\n\nCompute the logical "and" of two or more values. The `and_` command can be used as an infix operator after its first argument (`r.expr(True).and_(False)`) or given all of its arguments as parameters (`r.and_(True, False)`). The standard Python and operator, `&`, may also be used with ReQL.\n\n*Example* Return whether both `a` and `b` evaluate to true.\n\n    > a = True\n    > b = False\n    > (r.expr(a) & b).run(conn)\n    \n    False\n*Example* Return whether all of `x`, `y` and `z` evaluate to true.\n\n    > x = True\n    > y = True\n    > z = True\n    > r.and_(x, y, z).run(conn)\n    \n    True\n'),
```
Comment (danielmewes) [inline drivers/python/rethinkdb/docs.py]: This is also auto-generated from the files in This file is auto generated. These changes should be made in the corresponding files in https://github.com/rethinkdb/docs/tree/master/api

**Comment by eliangidoni** [comment]
@danielmewes  Ok. I've split this commit. See PR https://github.com/rethinkdb/docs/pull/934

**Comment by mlucy** [comment]
Thanks again for the pull request @eliangidoni !  This was never merged explicitly, but I made a very similar change by hand that fixed https://github.com/rethinkdb/rethinkdb/issues/4696.

**Pull request closed without merging**